lib/airbrake-ruby.rb

```diff
@@ -275,20 +275,16 @@ def create_deploy(deploy_params, notifier = :default)
     private
 
     ##
-    # Calls +method+ on +notifier+ with provided +args+.
+    # Calls +method+ on +notifier+ with provided +args+. If +notifier+ is not
+    # configured, returns an error hash.
     #
-    # @raise [Airbrake::Error] if none of the notifiers exist
+    # @return [Hash{String=>String}, Object]
     def call_notifier(notifier, method, *args, &block)
-      if @notifiers.key?(notifier)
-        @notifiers[notifier].__send__(method, *args, &block)
-      else
-        # If we raise SystemExit, the Ruby process can gracefully quit without
-        # the unwanted Airbrake::Error.
-        raise args.first if args.first.class == SystemExit
-
-        raise Airbrake::Error,
-              "the '#{notifier}' notifier isn't configured"
+      unless @notifiers.key?(notifier)
+        return { 'error' => "the '#{notifier}' notifier isn't configured" }
```

Comment:
How about: `**Airbrake: the '#{notifier}' notifier isn't configured, '#{method}' was not called.`.

This error can occur even if you call `add_filter`, for example, so if you call it and see `Airbrake will not be notified of errors`, it might be a little bit confusing. On the other hand, I doubt anybody will be checking the return value of `add_filter`. Thoughts?